mitmproxy/tools/main.py

```diff
@@ -3,7 +3,6 @@
 DO NOT use type annotations or other python3.6-only features that makes this file unparsable by older interpreters!
 """
 
-from __future__ import print_function  # this is here for the version check to work on Python 2.
```

Comment:
This might be debatable, but I think we're at the point where we can stop caring about py2 even here 👍 
(But please remove the comment above)

Reply:
I was actually going to put it back in - just because its not a real pain for us, and py27 EOL was just this year...